Update control-usb-devices-using-intune.md

msbemba opened the pull request:
Removed the Endpoint ATP baseline settings to control removable storage as Device control is no longer a part of baseline settings.

msbemba committed:
Update control-usb-devices-using-intune.md

Removed the Endpoint ATP baseline settings to control removable storage as Device control is no longer a part of baseline settings.

msbemba commented:
@JohanFreelancer9 Please copy review

msbemba commented:
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5458

JohanFreelancer9 reviewed (commented):
Should "Use Microsoft Defender for Endpoint baseline settings" not be removed from the preceding table as well?

msbemba committed:
Update control-usb-devices-using-intune.md

removed baseline settings from as Block or allow removable devices is not available in baseline settings

msbemba commented:
@JohanFreelancer9  Thank you. Could you please copy review again?

JohanFreelancer9 commented on microsoft-365/security/defender-endpoint/control-usb-devices-using-intune.md:
```diff
@@ -57,7 +57,7 @@ The following table describes the ways Microsoft Defender for Endpoint can allow
 | [Prevent installation and usage of specifically prohibited peripherals with matching device instance IDs](#prevent-installation-and-usage-of-specifically-prohibited-peripherals-with-matching-device-instance-ids) | You can't install or use prohibited peripherals that match any of these device instance IDs. |
 | [Limit services that use Bluetooth](#limit-services-that-use-bluetooth) | You can limit the services that can use Bluetooth. |
-| [Use Microsoft Defender for Endpoint baseline settings](#use-microsoft-defender-for-endpoint-baseline-settings) | You can set the recommended configuration for ATP by using the Defender for Endpoint security baseline. |
+
```
```suggestion
```

JohanFreelancer9 reviewed (changes requested):
Copy review.

msbemba committed:
Update microsoft-365/security/defender-endpoint/control-usb-devices-using-intune.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

yogkumgit commented:
@Dansimp Please merge this PR. Thanks

yogkumgit commented:
@Dansimp Gentle reminder

yogkumgit commented:
@Dansimp Gentle reminder.

chrisda committed:
Merge branch 'public' into patch-568
